packages/teleport/src/Databases/DatabaseList/DatabaseList.tsx

```diff
@@ -113,7 +116,19 @@ function DatabaseList(props: Props) {
           cell={<TextCell />}
         />
         <Column header={<Cell>Labels</Cell>} cell={<LabelCell />} />
+        <Column
+          header={<Cell />}
+          cell={<ConnectButton setDbConnectInfo={setDbConnectInfo} />}
+        />
       </StyledTable>
+      {dbConnectInfo && (
+        <ConnectDialog
+          user={user}
+          clusterId={clusterId}
+          dbConnectInfo={dbConnectInfo}
```

Comment:
I'd just pass it as individual props to avoid exporting and importing the dbConnectInfo type. I think the `ConnectDialog` importing the type is kind of weird b/c ConnectDialog should be the one dictating the type.